packages/dds/tree/src/simple-tree/proxyBinding.ts

```diff
@@ -3,97 +3,17 @@
  * Licensed under the MIT License.
  */
 
-import { assert } from "@fluidframework/core-utils/internal";
-
-import { type AnchorNode, type AnchorSet, type UpPath, anchorSlot } from "../core/index.js";
-import {
-	ContextSlot,
-	type FlexTreeNode,
-	assertFlexTreeEntityNotFreed,
-	flexTreeSlot,
-	type MapTreeNode,
-	isMapTreeNode,
-} from "../feature-libraries/index.js";
-import { fail } from "../util/index.js";
-import type { TreeNode } from "./core/index.js";
-// TODO: decide how to deal with dependencies on flex-tree implementation.
-// eslint-disable-next-line import/no-internal-modules
-import { makeTree } from "../feature-libraries/flex-tree/lazyNode.js";
+import type { InnerNode, TreeNode } from "./core/index.js";
 import { getKernel } from "./core/index.js";
 
 // This file contains various maps and helpers for supporting associating simple TreeNodes with their InnerNodes, and swapping those InnerNodes as part of hydration.
```

Comment:
This comment should be updated. However, perhaps this file should also be deleted? Is this helper really useful?

Reply:
There are 39 references to getOrCreateInnerNode, so I punted on dealing with it as part of this PR.

Reply:
Old file comment removed.